core/src/OpenMP/Kokkos_OpenMP_Parallel.hpp

```diff
@@ -49,1219 +33,8 @@ inline bool execute_in_serial(OpenMP const& space = OpenMP()) {
           !(omp_get_nested() && (omp_get_level() == 1)));
 }
```

Comment:
If that is all that is left in `<okkos_OpenMP_Parallel.hpp>` then that file needs a new name that reflects better what it provides.  An alternative (better?) would be to move this into `<Kokkos_OpenMP.hpp>` or whatnot.

Reply:
Moved this to `Kokkos_OpenMP_Instance.hpp`.